client/settings/payment-requests/payment-request-settings.js

```diff
@@ -15,43 +15,41 @@ import SettingsLayout from '../settings-layout';
 import LoadableSettingsSection from '../../components/loadable-settings-section';
 import SaveSettingsSection from '../save-settings-section';
 
-const paymentRequests = {
```

Comment:
Simplifying a bit - no need to create an object and destructure it later to get one of its properties. Just get the property itself